src/settings-ui/Settings.UI.Library/ViewModels/ColorPickerViewModel.cs

```diff
@@ -62,15 +66,14 @@ public ColorPickerViewModel(ISettingsUtils settingsUtils, ISettingsRepository<Ge
             GeneralSettingsConfig = settingsRepository.SettingsConfig;
 
             _settingsUtils = settingsUtils ?? throw new ArgumentNullException(nameof(settingsUtils));
-            if (_settingsUtils.SettingsExists(ColorPickerSettings.ModuleName))
-            {
-                _colorPickerSettings = _settingsUtils.GetSettingsOrDefault<ColorPickerSettings>(ColorPickerSettings.ModuleName);
```

Comment:
so, if this API is error-prone, perhaps we should delete it altogether?

Reply:
@yuyoyuppe should we create a new issue to get it removed?